styles/colors-silicon-day.less

```diff
@@ -298,17 +302,19 @@
 
 // Toggle Colors
 // ---------------------------------------
-@agate-togglebutton-indicator-off-bg-color: black;
-@agate-togglebutton-indicator-on-bg-color: @agate-highlight;
-@agate-togglebutton-indicator-off-border-color: @agate-button-border-color;
-@agate-togglebutton-indicator-on-border-color: @agate-accent;
+@agate-togglebutton-indicator-off-color: @silicon-dark-gray;
+@agate-togglebutton-indicator-on-color: @silicon-brand;
+@agate-togglebutton-indicator-off-bg-color: transparent;
+@agate-togglebutton-indicator-on-bg-color: transparent;
+@agate-togglebutton-indicator-off-border-color: transparent;
+@agate-togglebutton-indicator-on-border-color: transparent;
```

Comment:
Why do we need to define @agate-togglebutton-indicator-off-border-color and @agate-togglebutton-indicator-on-border-color in styles/colors-silicon-day.less because we already define them in styles/colors-base.less with the same values?